src/output-json-http.c

```diff
@@ -349,6 +351,55 @@ void JsonHttpLogJSONExtended(json_t *js, htp_tx_t *tx)
     json_object_set_new(js, "length", json_integer(tx->response_message_len));
 }
 
+static void HttpBodyToBuffer(HtpBody *body, MemBuffer *buffer)
+{
+    HtpBodyChunk *cur = body->first;
+    for ( ; cur != NULL; cur = cur->next) {
+        uint8_t *data = NULL;
+        uint32_t data_len = 0;
+        StreamingBufferSegmentGetData(body->sb, &cur->sbseg, (const uint8_t **)&data, &data_len);
```

Comment:
This is unnecessary.  You can just grab the single buffer w/o iterating the chunks.